lib/ControlledVocab/ControlledVocab.js

```diff
@@ -478,6 +478,8 @@ class ControlledVocab extends React.Component {
       user = <Link to={`/users/view/${metadata.updatedByUserId}`}>{name}</Link>;
     } else if (metadata.updatedByUserId === SYSTEM_USER_ID || !metadata.updatedByUserId) {
       user = <FormattedMessage id="stripes-smart-components.system" />;
+    } else {
+      user = <FormattedMessage id="stripes-smart-components.unknownUser" />;
```

Comment:
Alternatively, grab this from stcom instead: `stripes-components.metaSection.unknownUser` and then we don't need a new translation.